### PR TITLE
Comp add edit header

### PR DESCRIPTION
**Pull request by SomeViet**
Creating to subheader for Add and Edit pages

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,26 @@
 import { BrowserRouter, Switch, Route, Redirect } from "react-router-dom";
+import AddEditSubheader from "./components/AddEditSubheader/AddEditSubheader";
```
Comment (ndwilliamson123) [inline src/App.js:2]: We should add the AddEditSubheader component to the index.js file so we can import it on line 3

import { Footer, AddEditSubheader } from "./components/index";

**Review by ndwilliamson123 — CHANGES_REQUESTED** [review]
Code looks good other than the use of index.js

If someone can take a look at it visually, please do. Or I can review later tonight.

File: src/App.js (hunk 1 above)
Comment (SomeViet) [reply src/App.js]: Added component to index.js file

**Review by ndwilliamson123 — APPROVED** [review]
LGTM

**Pull request merged**